doc/nrf-bm/release_notes/release_notes_0.8.0.rst

```diff
@@ -0,0 +1,230 @@
+
+  * The ``bm_zms`` storage. TODO link when ready
+  * The ``Storage`` library.
+  * A Zephyr queue functionality.
+
+..
+
+* ``ble_adv`` library:
+
+  * Changed the default advertising name to ``nRF_BM_device``.
+  * Corrected the :kconfig:option:`BLE_ADV_DIRECTED_ADVERTISING_HIGH_DUTY` Kconfig option.
+
+..
+
+* ``ble_conn_params library`` library:
+
+  * Added:
+
+    * Support for asymmetric data length.
+    * The updated connection parameter values to the ``BLE_CONN_PARAMS_EVT_UPDATED``` event.
+
+  * Renamed slave latency to peripheral latency according to specification.
+
+  * Fixed:
+
+    * An issue where the ATT MTU could be set to a lower value than the current value.
+    * An issue where the list of PHYs supported by the SoftDevice was not respected by the library when doing PHY updates.
+    * Data length timeout issue and the :c:func:`data_length_get` function returning wrong value.
+    * Incorrectly stated maximum ATT MTU value.
+
+..
+
+* ``bm_buttons`` library:
+
+  * Added the use of nrfx critical section API.
+
+Samples
+=======
+
+* General changes:
+
+  * All samples now use the logging module instead of ``printk``.
+
+  * Updated:
+  
+    * The main loop sleep instructions as the :c:func:`sd_app_evt_wait` function is deprecated.
+    * Updated the main function flow to always end up in a loop with sleep instructions and deferred log processing instead of returning from the main function.
+      This ensures that log messages are printed and that the device goes into sleep mode.
+      Returning from main leaves the device in a state that prevents low power sleep and deferred log messages will be stored indefinitely in an internal buffer.
+
+Bluetooth samples
+-----------------
+
+* Added the :ref:`ble_pwr_profiling_sample` sample.
+
+..
+
+* :ref:`ble_cgms_sample` sample:
+
+  * Fixed an issue where a PHY update request was responded to twice.
+
+..
+
+* :ref:`ble_hrs_sample` sample:
+
+  * Removed the unused Coded PHY Kconfig fragment.
+
+..
+
+* :ref:`ble_nus_sample` sample:
+
+  * Added support for Low Power UARTE (LPUARTE).
+  * Fixed the pin configuration to use the pins from the ``board_config.h`` board header.
+
+Peripheral samples
+------------------
+
+* Added the following peripheral samples:
+
+  * :ref:`bm_zms_sample` sample.
+  * :ref:`bm_lpuarte_sample` sample.
+  * :ref:`storage_sample` sample.
+
+MCUboot samples
+---------------
+
+* Added the following samples:
+
+  * :ref:`ble_mcuboot_recovery_entry_sample` sample.
+  * :ref:`ble_mcuboot_recovery_retention_sample` sample.
+
+Known issues and limitations
+============================
+
+* Some issues are observed when using iPhone as the peer during testing of the Bluetooth samples.
+* Some issues are observed when using Linux with the Bluetooth Low Energy app in nRF Connect for Desktop as the peer during testing of the Bluetooth samples.
+* Bluetooth LE pairing and bonding is not supported.
+* The samples are not optimized for power consumption.
```

Comment:
Adjust to:
"The samples are not optimized for power consumption unless explicitly stated"